.appveyor.yml

```diff
@@ -0,0 +1,110 @@
+# AppVeyor.com is a Continuous Integration service to build and run tests under
+# Windows
+
+environment:
+  BUILD_COMMIT: v0.11.1
+  REPO_DIR: pyFFTW
+  PKG_NAME: pyFFTW
+  WHEELHOUSE_UPLOADER_USERNAME: travis-worker
+  # https://www.appveyor.com/docs/build-configuration/#secure-variables
+  # Encrypted to matthew-brett account, for now.
+  WHEELHOUSE_UPLOADER_SECRET:
+    secure: 9s0gdDGnNnTt7hvyNpn0/ZzOMGPdwPp2SewFTfGzYk7uI+rdAN9rFq2D1gAP4NQh
+  NP_BUILD_DEP: "numpy==1.10.4"
+  NP_TEST_DEP: "numpy==1.10.4"
+
+  # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
+  # /E:ON and /V:ON options are not enabled in the batch script intepreter
+  # See: http://stackoverflow.com/a/13751649/163740
+  CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\appveyor\\run_with_env.cmd"
+
+  matrix:
+    - PYTHON: C:\Python27-x64
+      PYTHON_ARCH: "64"  # define this to download 64-bit FFTW dlls
+      PYTHON_VERSION: "2.7"  # used by run_with_env.cmd
+    - PYTHON: C:\Python35-x64
+      PYTHON_ARCH: "64"
+      PYTHON_VERSION: "3.5"
+    - PYTHON: C:\Python36-x64
+      PYTHON_ARCH: "64"
+      PYTHON_VERSION: "3.6"
+      NP_BUILD_DEP: "numpy==1.12"
+      NP_TEST_DEP: "numpy==1.12"
+    - PYTHON: C:\Python37-x64
+      PYTHON_ARCH: "64"
+      PYTHON_VERSION: "3.7"
+      NP_BUILD_DEP: "numpy==1.14.5"
+      NP_TEST_DEP: "numpy==1.14.5"
+
+init:
+  - "ECHO \"%APPVEYOR_SCHEDULED_BUILD%\""
+
+install:
+
+  # Prepend newly installed Python to the PATH of this build (this cannot be
+  # done from inside the powershell script as it would require to restart
+  # the parent CMD process).
+  - SET PATH=%PYTHON%;%PYTHON%\Scripts;%PATH%
+
+  # Check that we have the expected version and architecture for Python
+  - python --version
+  - python -c "import struct; print(struct.calcsize('P') * 8)"
```

Comment:
Should this come with some kind of assert or similar to break the build if it is wrong or is this merely for visible inspection?

Reply:
I was just using it visually when debugging this to make sure it was as expected